public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx

```diff
@@ -379,6 +377,8 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
         .setTitle('')
         .setOption('showTypeIcons', true)
         .setOption('showHeader', true)
+        // this needs to use same instance as the queries tab is subscribing to it
+        .setData(this.queryRunner)
```

Comment:
@torkelo, this will change the parent of the query runner. It should be fine as we are in the panel edit context, so the parent should not really matter.  But maybe it would be worth mentioning it.

But the main problematic part here is that the table view should get the `SceneDataTransformer`, otherwise it shows the non-trasnfromed query results. In the old arch the table view respects transformations.

Reply:
@dprokop good catch! pushed a fix for that